docs/src/content/docs/guides/css-and-tailwind.mdx

```diff
@@ -50,16 +50,28 @@ Customize the styles applied to your Starlight site by providing additional CSS
 
 You can see all the CSS custom properties used by Starlight that you can set to customize your site in the [`props.css` file on GitHub](https://github.com/withastro/starlight/blob/main/packages/starlight/style/props.css).
 
+### Cascade layers
+
+Starlight uses [cascade layers](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers) internally so that any custom unlayered CSS will override the default styles.
```

Comment:
Small suggestion: This part can be slightly expanded to mention the main purpose of using layers and to include the `starlight` layer name.

```suggestion
Starlight uses [cascade layers](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers) internally for managing the order of styles. This results in a more predictable CSS order and allows for easier overrides. Any custom unlayered CSS will override the default styles from the `starlight` layer.
```